client/src/components/friends.jsx

```diff
@@ -0,0 +1,42 @@
+import {
```

Comment:
fyi - I renamed `friends.js` to `friends.jsx` so Git thinks this is a whole new file even though I only changed a few lines.